build.gradle

```diff
@@ -238,6 +239,7 @@ dependencies {
 
     testCompile project(':groovy-ant')
     testCompile project(':groovy-test')
+    testCompile project(':groovy-macro')
```

Comment:
Adding this line alone is enough to cause the test failures. We obviously still need to determine why but at least we know your others changes are causing the problem.